apps/site/src/lib/components/Accordion/Accordion.module.scss

```diff
@@ -78,3 +82,11 @@
 		animation: slideUp 300ms cubic-bezier(0.87, 0, 0.13, 1);
 	}
 }
+
+@media (min-width: 640px) {
+	.content {
+		p {
+			font-size: 1.25rem;
+		}
+	}
```

Comment:
Nitpick: You should just be able to say `.content p` and use only one set of curly braces instead of having it nested like this.